### PR TITLE
Added the v2.0.0 => v1.0.0 converter

### PR DESCRIPTION
**Pull request by czardoz**
Do not merge this PR yet, tests are yet to be added. For now, I've manually tested by importing a converted collection into the app.

### CODE REVIEW

File: bin/transform-collection.js [hunk 1]
```diff
@@ -20,12 +20,17 @@ program
     .action(function (options) {
         transformer.convert(options, function (error, result) {
             if (error) {
-                console.log(error);
+                console.error(error.message);
```
Comment (shamasis) [inline bin/transform-collection.js]: Some error might not have `.message`. A good precaution would be `error.message || error`.

File: lib/converters/converter-v2-to-v1.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+module.exports = {
+    input: '2.0.0',
+    output: '1.0.0',
+    builders: builders,
+
+    convert: function (collection, options, callback) {
+        var units = ['order', 'folders', 'requests'],
+            newCollection = {
+                id: collection.info.id,
```
Comment (shamasis) [inline lib/converters/converter-v2-to-v1.js]: Is `info` guaranteed to exist? Will the conversion fail gracefully if `info` is absent?

**Comment by shamasis** [comment]
@czardoz  The code is merged... but needs some more tweaks and error trapping.

File: lib/converters/converter-v2-to-v1.js (hunk 2 above)
Comment (czardoz) [inline lib/converters/converter-v2-to-v1.js]: Yes, info is guarenteed to exist, it contains the collection name, and ID.

File: bin/transform-collection.js (hunk 1 above)
Comment (czardoz) [inline bin/transform-collection.js]: I've fixed this in PR #5

**Pull request merged**